app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -113,6 +113,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     if (target.getAttribute('data-ga4-index')) {
       try {
         index = JSON.parse(target.getAttribute('data-ga4-index'))
+
+        // Check whether the index object already exists on a parent element, as is the case with tracking accordion links.
+        // If true, combine data-ga4-index with the index object
+        if (typeof indexData === 'object') {
```

Comment:
Could this be done when the indexes are first set (the `setIndexes` function) instead? Just because we set create `data-ga4-index` ourselves, but then it seems like we're fixing it here, when we have control over both functions. Or would doing it in there instead affect other components?

Reply:
Potentially yeah, I'll need to have a quick think about it